db/schema.rb

```diff
@@ -27,6 +27,8 @@
   end
 
   create_table "race_entries", force: :cascade do |t|
+    t.integer "racer_id"
+    t.integer "race_edition_id"
```

Comment:
These were added automatically when I ran migrations (because I ran them after the correct relationships were added to the model).